src/subcommand/wallet/pending.rs

```diff
@@ -0,0 +1,20 @@
+use {super::*, crate::wallet::entry::EtchingEntry};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct PendingOutput {
+  pub etchings: Vec<EtchingEntry>,
+}
+#[derive(Debug, Parser)]
+pub(crate) struct Pending {}
+
+impl Pending {
+  pub(crate) fn run(self, wallet: Wallet) -> SubcommandResult {
+    let etchings = wallet
+      .pending_etchings()?
+      .into_iter()
+      .map(|(_, entry)| entry)
```

Comment:
I'm just returning the entire etching entry, making the output quite verbose. Please, let me know if another output schema is more desirable.